app/views/shared/_product_view.erb

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Just a suggestion, but I might add an if to only show the flash notice paragraph tags if they're there.   Also if you're using the row for the grid, it's nicer to also include an inside div that specifies the columns, just to deal with possible side effects.  

```
<% if flash[:notice] %>
   <div class="row">
      <div class="column small-12">
        <p> <%= flash[:notice] %></p>
      </div>
    </div>
<% end %>
```